### DISCUSSION TITLE
Ext Remover

### DISCUSSION THREAD

**Discussion by Reloadandtryagain** [3kh0/ext-remover#145] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
I accidentally updated chrome and now i can't use the thing. Could you patch it up or help. I can't downgrade my update from 106 it's saying unknown command: rm

**Comment by 3kh0** [comment, marked as answer]
@userisnotavalible make sure you follow each intruction, send screenshots
